### ISSUE TITLE
db.connect is not working

### ISSUE THREAD

**Issue by kingabzpro** [v1a0/sqllex#9]
```python
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-20-23b07dc9b487> in <module>
      1 db = SQLite3x('/work/Database/database.sqlite/database.sqlite')
      2 
----> 3 db.connect()

AttributeError: 'SQLite3x' object has no attribute 'connect'

```

**Comment by v1a0** [comment]
db.connect() will be available in SQLLEX v0.1.8+ 

You can clone repo from github or wait till I release an new version

I'll release v0.1.8.2 in 10 minutes

**Comment by v1a0** [comment]
@kingabzpro! SQLLEX v0.1.8.2 has released, please upgrade your package for current version

```shell
pip install sqllex --upgrade
```

And let me know is this issue fixed or not
